azurecompute-arm/src/main/java/org/jclouds/azurecompute/arm/config/AzureComputeHttpApiModule.java

```diff
@@ -46,9 +44,9 @@ protected void bindErrorHandlers() {
    @Override
    protected void installLocations() {
       super.installLocations();
-      bind(ImplicitLocationSupplier.class).
-              to(OnlyLocationOrFirstRegionOptionallyMatchingRegionId.class).
```

Comment:
This was working before. If you discard the changes in the location transformation function, can't you just discard the changes in this class too?